docs/root/configuration/http/http_filters/adaptive_concurrency_filter.rst

```diff
@@ -192,10 +192,21 @@ The adaptive concurrency filter outputs statistics in the
 comes from the owning HTTP connection manager. Statistics are specific to the concurrency
 controllers.
 
+Adaptive Concurrency Statistics
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The adaptive concurrency filter uses the namespace
+*http.<stat_prefix>.adaptive_concurrency.*.
+
+.. csv-table::
+  :header: Name, Type, Description
+  :widths: auto
+
+  enabled, Gauge, Set to 1 if the filter is enabled and 0 otherwise.
```

Comment:
This is not something that we really do anywhere else. Is this really needed? Can't whether it's enabled per inferred from other stats?